APE1.rst

```diff
@@ -139,6 +139,28 @@ An APE's status can
   rationales, so that new community members can understand why a decision was
   made.
 
+Modifications to an APE
+.......................
+
+**Addendum: July 22, 2020**
+
+An APE can be modified, with one person acting as "champion" for the
```

Comment:
Champion is already introduced above and called author later. I think keeping that language is better than redoing it.

Reply:
so ... remove the quotation marks?

Reply:
Yes, I'd say to just take out the quotation marks.

Reply:
```suggestion
An APE can be modified, with one person acting as champion for the
```
(to make it easier to do right here)

Reply:
Done